htdocs/comm/propal/card.php

```diff
@@ -2579,7 +2579,7 @@
 				}
 
 				// Create an invoice and classify billed
-				if ($object->statut == Propal::STATUS_SIGNED) {
+				if ($object->statut == Propal::STATUS_SIGNED && empty($conf->global->PROPAL_NOT_BILLABLE)) {
```

Comment:
For this new constant, ca you use PROPOSAL_ARE_NOT_BILLABLE  ?

Reply:
@eldy  ok; it is done now.